iot-e2e-tests/common/pom.xml

```diff
@@ -84,24 +84,22 @@
             <artifactId>jmockit</artifactId>
             <version>1.24</version>
         </dependency>
-        <!-- https://mvnrepository.com/artifact/org.littleshoot/littleproxy -->
-        <dependency>
-            <groupId>org.littleshoot</groupId>
-            <artifactId>littleproxy</artifactId>
```

Comment:
This dependency isn't being maintained anymore, so I'm porting the code into our repo so that we can upgrade it as necessary. In particular, this library took a dependency on an older guava version and we couldn't simply force it to use a newer guava version without code changes